packages/core/src/lib/markdown-it/highlight/HighlightRule.js

```diff
@@ -11,8 +9,36 @@ class HighlightRule {
     this.ruleComponents = ruleComponents;
   }
 
+  static parseAllRules(allRules, lineOffset, tokenContent) {
+    const highlightLines = this.splitByChar(allRules, ',');
+    const strArray = tokenContent.split('\n');
+    return highlightLines
+      .map(ruleStr => HighlightRule.parseRule(ruleStr, lineOffset, strArray))
+      .filter(rule => rule); // discards invalid rules
+  }
+
+  static splitByChar(allRules, splitter) {
+    const highlightRules = [];
+    let isWithinQuote = false;
+    let currentPosition = 0;
+    for (let i = 0; i < allRules.length; i += 1) {
```

Comment:
The convention for `for` loops in MarkBind seem to be using `i++` instead of `i += 1` when incrementing by 1 (as far as I can see)!

Reply:
It seems like its mostly += 1 when I ctrl F and Eslint is giving me an error when ```i++``` is used. Shouldn't it be += 1?